content/docs/command-reference/status.md

```diff
@@ -85,12 +85,18 @@ file name or hash is shown, along with a _state description_, as detailed below:
   storage.
 - _deleted_ means that the file/directory doesn't exist in the cache, but exists
   in remote storage.
+- _missing_ means that the file/directory doesn't exist neither in cache,
+  nor in remote storage.
 
-For either _new_ and _deleted_ data, the cache (subset determined by the current
-workspace) is different from remote storage. Bringing the two into sync requires
+For _new_ and _deleted_ data, the cache is different from remote storage.
+Bringing the two into sync requires
 `dvc pull` or `dvc push`. For the typical process to update the workspace, see
 [Sharing Data And Model Files](/doc/use-cases/sharing-data-and-model-files).
 
+For _missing_ data both cache and remote storage are out of date. Probably
+someone on another machine didn't execute `dvc push` after adding new data. You
+should run `dvc add` or `dvc repro` to rid off _missing_ state.
```

Comment:
The intro here is repetitive with the description in the bullet above. Also, I don't think `dvc add` wouldn help you here. Suggestion:

```suggestion
For _missing_ data, there's nothing to retrieve from storage. This can happen
for example in fresh <abbr>DVC repository</abbr> clones if the data wasn't
uploaded from the original repo, or after certain uses of `dvc gc`. You can try
`dvc repro` to regenerate the output locally, and `dvc push` remotely after that.
```
> Paragraph may needs formatting

Reply:
OK this is the only one left. I'll just commit it and merge the restyled PR to save you time, Nik.